api/api.go

```diff
@@ -30,6 +30,7 @@ import (
 	"strconv"
 
 	signalstrength "github.com/TheCacophonyProject/management-interface/signal-strength"
+	goAPI "github.com/gferraro/go-api"
```

Comment:
In idomatic Go, import aliases should just be lower case - so `goapi` in this case